### PR TITLE
different types of buttons made

### PR DESCRIPTION
**Pull request by Asypher00**
I have made 4 buttons. Primary Button , Link Button , Icon Button and Floating Action Button
Screenshot (5)
.

### CODE REVIEW

File: Button.css [hunk 1]
```diff
@@ -0,0 +1,36 @@
+.btn {
+    margin : 1.25rem;
+    font-family: "comic sans ms";
+    font-weight: 50%;
+    width : 12.5rem;
+    height : 3.125rem;
+    font-size: 1.125rem;
+    background-color: aqua;
+
```
Comment (tanishq20) [inline Button.css:9]: Avoid giving extra spaces.

File: Button.css [hunk 2]
```diff
@@ -0,0 +1,36 @@
...
+    height : 3.125rem;
+    font-size: 1.125rem;
+    background-color: aqua;
+    
+    border : 0.125rem solid white;
+    border-radius: 1.25rem;
+
+}
+
+.txt-style{
```
Comment (tanishq20) [inline Button.css:15]: You can move this txt-style to your utility class, as it will be used in multiple places.

File: Button.css [hunk 3]
```diff
@@ -0,0 +1,36 @@
...
+    background-color: white;
+    border : none;
+}
+
+.btn-icon{
+    background-color: white;
+    border : none;
+}
+
+.btn-float{
```
Comment (tanishq20) [inline Button.css:30]: You can give some padding in float btn. It will increase visibility.

File: Button.css [hunk 4]
```diff
@@ -0,0 +1,36 @@
...
+    border-radius: 1.25rem;
+
+}
+
+.txt-style{
+    color : rgb(0, 0, 0);
+    text-decoration: none;
+}
+
+.btn-link {
```
Comment (krishanu2911) [inline Button.css:20]: Just a suggestion like you can pass this border: none and text-decoration: none properties to the reset CSS which will be applied to all button and a tag then you don't have to specify every time.
Everything else is good to go.

Comment (Asypher00) [reply Button.css:20]: noted

**Pull request merged**